### ISSUE TITLE
Presentation Creation & Guidelines

### ISSUE THREAD

**Issue by Serubin** [RITlug/tasks#2]
This issue will contain information on our weekly presentation/workshop tasking and process.

## Guidelines 

Link to our default presentation template. https://goo.gl/1dqinL

<img width="512" alt="image" src="https://user-images.githubusercontent.com/1234465/29251911-bce1c61a-802b-11e7-9156-064ae0cb3b56.png">

Things to include:
* Overviews/Agendas

**Comment by jflory7** [comment]
@Serubin Is the goal with this one to create a deliverable presentation template that we can share with members and also guidelines on how to put one together?

**Comment by Serubin** [comment]
@jflory7 My aim is to have a basic template that we can use to speed things up. If nothing else, just a default opening page.

The above works well as that and can be modified as needed.

**Comment by jflory7** [comment]
@Serubin What work is remaining on this task?

**Comment by Serubin** [comment]
Nothing - this is just open for reference

**Comment by jflory7** [comment]
@Serubin Is there anything we can do to close this ticket? What kind of deliverables do we want to produce here?

**Comment by axk4545** [comment]
@Serubin @jflory7 is  the template in? if so we can add this to runbook and close.

**Comment by jflory7** [comment]
@axk4545 I don't know of any template we have. @Serubin needs to weigh in here. If we don't have the bandwidth for the template, I think we should close the issue since it's not a high-priority item for the club.

**Comment by axk4545** [comment]
@jflory7 kinda[ https://goo.gl/1dqinL](https://goo.gl/1dqinL)

**Comment by jflory7** [comment]
@axk4545 Seems to me like it's mostly our logo plastered on the opening slide. If we still want to do this, I see two tasks:

1. Add more themed slides (section slide, media slides, etc.)
2. Make slides accessible (either by download on site or way to "import" from Google Docs)

If @Serubin has the bandwidth for this, he can take it on – otherwise, I think we can close for lack of bandwidth.

**Comment by Serubin** [comment]
@jflory7 The link above is the template. It should be added to the runbook now it's updated.

**Comment by Tjzabel** [comment]
@Serubin @jflory7 Where is the link to the slides template in the runbook? If the template has been added, we can close this issue.

Otherwise, we can leave this issue open as a reference to any outside person looking to have a template available to use for a presentation.

**Comment by jflory7** [comment]
Getting this in the Runbook is blocked by a design issue. We need to style it with the [new logos](https://github.com/RITlug/ritlug.github.io/tree/master/img). Could someone update the default template? I will work on getting it into the Runbook after.

**Comment by ct-martin** [comment]
@jflory7 This is on the eboard agenda along with the poster redesign

**Comment by jflory7** [comment]
The RITlug brand (including posters) now lives in [RITlug/brand](https://github.com/RITlug/brand).

The poster redesign work is superseded by #28. @ct-martin Thank you for filing that.

I'm closing this ticket as complete since I think we were confused on what the next step was.
